### PR TITLE
React App Setup

### PR DESCRIPTION
**Pull request by smilegupta**

### CODE REVIEW

File: .huskyrc.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "husky": {
+      "hooks": {
+        "pre-commit": "lint-staged",
+        "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+        "prepare-commit-msg": "exec < /dev/tty && git cz --hook || true"
+               }
+            }
```
Comment (anshuraj) [inline .huskyrc.json:8]: fix json syntax

File: src/Common/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Sidebar = () => {
+  return (
+        <div>
+            I am sidebar
+        </div>
```
Comment (anshuraj) [inline src/Common/Sidebar.js:7]: mixed indentation

**Review by anshuraj — COMMENTED** [review]
Found few indentation issues across multiple files. Add a new line to the end of every file. Add `.DS_Store` to gitignore

File: .huskyrc.json (hunk 1 above)
Comment (InvincibleZeal) [reply .huskyrc.json:8]: It's fixed in the other PR. I'll merge it onto this one.

File: src/Common/Sidebar.js (hunk 2 above)
Comment (InvincibleZeal) [reply src/Common/Sidebar.js:7]: This is also fixed. Linting wasn't working correctly in this PR.

**Review by anshuraj — APPROVED** [review]
LGTM

**Pull request merged**